.travis.yml

```diff
@@ -8,18 +8,14 @@ cache:
 install:
   - pip3 install -r requirements.txt
 
-script:
-  - py.test -v --color=yes --cov=.
-
-after_success:
-  - codecov
-
-deploy:
-  skip_cleanup: true
-  provider: script
-  script: bash etc/deploy_docs.sh
-  on:
-    branch: development
+jobs:
+  include:
+    - stage: compile
+      script: python3.5 -m compileall lib/
+    - stage: test
+      script: py.test -v --color=yes --cov=.
+    - script: sleep 10   # for unit tests in future
+    - script: sleep 10   # for unit tests in future
 
```

Comment:
We need a third pipeline stage named **docs** with the following code.
```bash
-  provider: script		
 -  script: bash etc/deploy_docs.sh		
 -  on:		
 -    branch: development
```
Please put this in correct yaml format.